puppet/check_puppet.rb

```diff
@@ -17,7 +17,8 @@
 require 'optparse'
 require 'yaml'
 
-statedir = "/var/lib/puppet/state"
+puppet_maj_version = `puppet --version`[0].to_i
```

Comment:
Thanks for this!

small nit pick here, but can you do this instead:

```
Integer(`puppet --version`.split(".")[0])
```

Few reasons:

  * We hope we don't get to Puppet 10 but what if? They might change their scheme
  * `"a".to_i` is 0 while `Integer("a")` raises an error, I think should something bad happens you rather want to raise

Really we should probably even check that the *``* exited 0 to be safe but probably in that case other things will go wrong